koga

```diff
@@ -122,7 +123,8 @@
   ;; Do the absolute minimum to inform ASDF about the location of systems
   ;; in order to find the clasp root and the desired build directory.
   (asdf:initialize-source-registry
-    `(:source-registry (:tree ,(uiop:getcwd))
+    `(:source-registry (:also-exclude "asdf")
```

Comment:
Maybe should mention why asdf is excluded in a comment (i guess to avoid reloading it from an implementation asdf?)

Reply:
Yeah, it is excluded so the host uses their own. Mostly because ECL doesn't like the latest ASDF.